assets/src/edit-story/elements/image/edit.js

```diff
@@ -41,25 +48,176 @@ const FadedImg = styled.img`
 	${ ImageWithScale }
 `;
 
-const ActualImg = styled.img`
+const CropImg = styled.img`
 	position: absolute;
 	${ ImageWithScale }
 `;
 
-function ImageEdit( { src, origRatio, width, height, x, y, scale, focalX, focalY, rotationAngle, isFullbleed } ) {
+function ImageEdit( { id, src, origRatio, width, height, x, y, scale, focalX, focalY, rotationAngle, isFullbleed } ) {
+	const [ fullImage, setFullImage ] = useState( null );
+	const [ croppedImage, setCroppedImage ] = useState( null );
+	const [ cropBox, setCropBox ] = useState( null );
+
+	const { actions: { setPropertiesById } } = useStory();
+
 	const elementProps = getBox( { x, y, width, height, rotationAngle, isFullbleed } );
 	const imgProps = getImgProps( elementProps.width, elementProps.height, scale, focalX, focalY, origRatio );
+
+	const resizeMoveableRef = useRef();
+	const panMoveableRef = useRef();
+	const translateRef = useRef( [ 0, 0 ] );
+	const cropRef = useRef( [ 0, 0, 0, 0 ] );
+
+	const updatePan = () => {
+		const [ tx, ty ] = translateRef.current;
+		fullImage.style.transform = `translate(${ tx }px, ${ ty }px)`;
+		croppedImage.style.transform = `translate(${ tx }px, ${ ty }px)`;
+	};
+
+	// Refresh moveables to ensure that the selection rect is always correct.
+	useEffect( () => {
+		if ( resizeMoveableRef.current ) {
+			resizeMoveableRef.current.updateRect();
+		}
+		if ( panMoveableRef.current ) {
+			panMoveableRef.current.updateRect();
+		}
+	} );
+
 	return (
 		<Element { ...elementProps }>
-			<FadedImg src={ src } { ...imgProps } />
-			<ActualBox>
-				<ActualImg src={ src } { ...imgProps } />
-			</ActualBox>
+			<FadedImg ref={ setFullImage } draggable={ false } src={ src } { ...imgProps } />
+			<CropBox ref={ setCropBox }>
+				<CropImg ref={ setCroppedImage } draggable={ false } src={ src } { ...imgProps } />
+			</CropBox>
+
+			{ /* Resizable moveable for cropping */ }
+			{ ! isFullbleed && (
+				<Movable
+					ref={ resizeMoveableRef }
+					className="crop-movable"
+					targets={ cropBox }
+
+					origin={ false }
+					edge={ false }
+
+					draggable={ false }
+					scalable={ false }
+					rotatable={ false }
+					warpable={ false }
+					pinchable={ false }
```

Comment:
^ These should be `false` by default mostly, I think.

Reply:
done